### ISSUE TITLE
Some icons are missing

### ISSUE THREAD

**Issue by anhng195** [celluloid-player/celluloid#854]
Overview Description: 
Some icons are missing.
![image](https://i.ibb.co/Wv7bQ52/Screenshot-from-2023-04-07-18-38-42.png)

Version: 0.25

Additional Information:
I installed it via Flatpak on Fedora 37

**Comment by CapeCrusader321** [comment]
Its probably a theming issue not a celluloid issue i believe, which icon theme are you using ??

**Comment by anhng195** [comment]
@CapeCrusader321 You're right, I fixed it by using this command `flatpak override --system --filesystem=$HOME/.icons/`
